tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist = py{27, 36}
+
+[testenv]
+passenv = test_flags
+commands = python setup.py nosetests --xunit-file=nosetests_{envname}.xml {env:test_flags}
```

Comment:
@ajoubertza do you think it's ideal to update tox to match -> https://github.com/ska-sa/katlogger/blob/master/tox.ini on this PR? Or that will form part of refactoring after this pr is merged?

Reply:
You can update it in this PR.

Reply:
![image](https://user-images.githubusercontent.com/7910856/67415034-811bf900-f5c4-11e9-80bd-df5fcae78b26.png)

Coverage @ 91% :+1: